core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -134,7 +136,10 @@ public void init() throws StoreInitializationException {
               org.apache.kafka.common.serialization.ByteArraySerializer.class);
     props.put(ProducerConfig.RETRIES_CONFIG, 0); // Producer should not retry
 
+    props.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG,
+            this.config.getString(SchemaRegistryConfig.KAFKASTORE_SECURITY_PROTOCOL_CONFIG));
     addSslConfigsToClientProperties(this.config, props);
+    addSaslConfigsToClientProperties(this.config, props);
```

Comment:
Would it make sense to have a `addSecurityConfigsToClientProperties`? This code also exists in at least one other place.